drush/Commands/DeployCommands.php

```diff
@@ -71,7 +71,7 @@ public function backstop(string $target, string $reference, array $options = ['c
     $stack = $this->getStack();
     $client = new \GuzzleHttp\Client(['handler' => $stack]);
     $options = [
-      'auth' => [$this->getTokenCircle()],
+      'auth' => [$this->getTokenCircle(), ''],
```

Comment:
We were getting a warning from Guzzle about not having a second argument, since auth expects a username and password rather than a token. Alternatively, this could be added as an Authorization header, but this change resolves the issue as well.